File: jprogram/even.java
import java.io.*;

import java.util.*;

class Main {

  public static void main(String[] args) {
 
   int Counter, Number;
   
 Scanner object = new Scanner(System.in);
 
   System.out.print("Enter the number to print the even numbers upto the number:");

    Number = object.nextInt();
    
System.out.print("The even numbers upto " + Number + " are");

    for(Counter = 0; Counter <= Number; Counter++) {

      if(Counter % 2 == 0) {
  
      System.out.print(" " + Counter);
    
  }
  
  }
 
   System.out.print(".");

  }
}

File: Csharp/even.cs
using System;


class MainClass {

  public static void Main (string[] args) {

    Console.Write("Enter the number to print even numbers upto the given number:");
   
 int Count = Convert.ToInt32(Console.ReadLine());

    Console.Write("The even numbers upto {0} are", Count);

    for(int Counter = 0; Counter <= Count; Counter++) {
    
  if(Counter % 2 == 0) {
  
      Console.Write(" {0}", Counter);
     
 }
    }
   
 Console.Write(".");
  
}
  }